xml/deploy_cephadm.xml

```diff
@@ -531,24 +533,51 @@ o- ssh ................................................... [Key Pair set]
    <sect3 xml:id="deploy-cephadm-configure-ntp">
     <title>Configure Time Server</title>
     <para>
-     Select one of the &sminion;s to be a time server for the rest of the
-     cluster, and configure it to synchronize its time with a reliable time
-     source outside of the cluster.
+     All cluster hosts need to have their time synchronized with a reliable
+     time source. There are several scenarios to approach time synchronization:
     </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       If all cluster hosts are already configured to synchronize their time
+       using a NTP service of choice, disable time server handling completely:
+      </para>
+<screen>
+&prompt.smaster;ceph-salt config /time_server disable
+</screen>
+     </listitem>
+     <listitem>
+      <para>
+       If your site already has a single source of time, specify the external
+       host name of the time source:
+      </para>
+<screen>
+ &prompt.smaster;ceph-salt config /time_server/server_hostname set <replaceable>pool.ntp.org</replaceable>
```

Comment:
`pool.ntp.org` is already being used for "external time servers", maybe we should use something that makes it clear that this is an "internal" host?